obulaworld/my-diary
# feat(fetch-api) Consume post entry api

**What does this PR do?**
Builds out the get post entry fetch function to consume POST /entries api

**Description of Task to be completed?**

- Create a AddEntry.js file
- Build out fetch function for the POST /entries url

**How should this be manually tested?**

Clone the repo,
Cd into feat/#159428721/user-should-be-able-consume-create-api
Run npm install
Run npm test
**Any background context you want to provide?**
No

**What are the relevant pivotal tracker stories?**
#159428721
